### DISCUSSION TITLE
New firmware 3.90 - breaking changes?

### DISCUSSION THREAD

**Discussion by Crashman1983** [BenPru/luxtronik#275] [General]
Yesterday AI released a new firmware version 3.90. According to some reports, the current luxtronic does not work with the new firmware. 

The web interface seems to be not available any more.

do you guys know more about that?

**Comment by wiljums** [comment]
you have to turn on the webinterface in the menu of the heatpump

**Comment by wiljums** [comment]
with this new firmware it is now possible to control the menu of the heatpump through the webinterface
![chrome_YsquNmby4L](https://github.com/user-attachments/assets/7b5b3106-b8ba-4978-9121-8b34f3a61834)

**Comment by blue-bean** [comment]
Controlling the menu/display by webbrowser is possible since lot of versions before...

**Comment by lancer73** [comment]
@Crashman1983 Do you have any sources for those reports?

@wiljums Can I conclude you have updated to 3.90 and have no problems?

**Comment by wiljums** [comment]
> @wiljums Can I conclude you have updated to 3.90 and have no problems?

correct, I have a SWCV92K3

**Comment by lancer73** [comment]
I can confirm I have no problems after the update. Had to correct the time, but that was it.

Alpha Innotec LWDV

**Comment by BenPru** [comment]
Okay, you Report that the Interface ist working normal to use this integration.
Does anyone know the changes of this Firmware?
Any new features or just fixes?
For my HP Novelan LAD7 3.88 ist the newest Firmware.

**Comment by lancer73** [comment]
Haven't seen anything new yet, except for a split in enabling the webserver at port 80 and the API at other ports. 

After update the webserver is by default disabled.

Also, the firmware image schrank about 300 - 400k. I suspect some cleaning up and/or refactoring.

**Comment by blue-bean** [comment]
> ...Does anyone know the changes of this Firmware? Any new features or just fixes? ...

The main new feature is the power limitation (see also https://github.com/BenPru/luxtronik/issues/267 ).

**Comment by BenPru** [comment]
@Crashman1983 
Can we close this issue?
